src/rgw/rgw_rest_user_policy.cc

```diff
@@ -142,15 +143,32 @@ void RGWPutUserPolicy::execute(optional_yield y)
 
   try {
     const Policy p(s->cct, s->user->get_tenant(), bl);
+    uint32_t pol_size_occupied = user->get_user_policy_size();
+    uint32_t pol_size = policy.size();
+
     map<string, string> policies;
     if (auto it = user->get_attrs().find(RGW_ATTR_USER_POLICY); it != user->get_attrs().end()) {
       bufferlist out_bl = it->second;
```

Comment:
if by any chance we don't want to store the total size of policies per user, we can use `out_bl.length()` which will be the total size of the map.
Storing size of each document is more accurate though.